torch/utils/_sympy/interp.py

```diff
@@ -101,7 +102,11 @@ def handlers():
         Identity: "identity",
         IsNonOverlappingAndDenseIndicator: "is_non_overlapping_and_dense_indicator",
         RoundDecimal: "round_decimal",
+        # TODO: do the rest of the opaque unary functions...
+        OpaqueUnaryFn_log2: "log2",
```

Comment:
I don't want to use the sympy functions because they are too smart and try to do simplifications but I don't want simplifications I want my float functions as is NO fancypants